packages/portal/src/components/set/AddItemToSetModal.vue

```diff
@@ -16,7 +16,10 @@
     >
       {{ $t('set.actions.createNew') }}
     </b-button>
-    <div class="collections">
+    <div
+      v-if="collections"
```

Comment:
This could still be rendered when collections is an empty array, which is truthy.

Reply:
OK. Reverted.